packages/backend/src/models/flow.ts

```diff
@@ -133,6 +134,10 @@ class Flow extends Base {
     })
   }
 
+  async getUser(): Promise<User> {
```

Comment:
Why do we need this extra function instead of just calling it directly in action.ts?

Reply:
nice thankss